app/controllers/citizens_controller.rb

```diff
@@ -14,7 +15,7 @@ def create
     @citizen = Citizen.new(permitted_params)
     respond_to do |format|
       if @citizen.save
-        CitizenMailer.notify_new_citizen(@citizen).deliver_later
+        CitizenMailer.notify_new_citizen(@citizen.id).deliver_later
```

Comment:
Excelente! Foi aquilo que conversamos em call. Aqui tem mais detalhes para um contexto melhor:

https://github.com/mperham/sidekiq/wiki/Best-Practices#2-make-your-job-idempotent-and-transactional